### DISCUSSION TITLE
montage tiles starting east

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#4688] [Digital Image Processing]
I have tiles 1.jpg 2.jpg 3.jpg 4.jpg 

I want to put them into 5x1 tiles starting east. <Blank> 4.jpg 3.jpg 2.jpg 1.jpg.

Is there a way to do this?

Thanks

**Comment by srini8080** [comment]
-reverse will not work as I want the first tile blank

**Comment by snibgo** [comment]
I'm not sure what you want. Perhaps:
```
magick montage 1.jpg 2.jpg 3.jpg 4.jpg -flop -tile 5x1 miff:- | magick - -flop out.png
```

**Comment by srini8080** [comment]
Fabulous!. This is what I wanted
